e2e/pages/caseList.page.js

```diff
@@ -43,12 +43,13 @@ module.exports = {
     I.click(this.fields.search);
   },
 
-  searchForCasesWithName(caseName, state='Any') {
+  async searchForCasesWithName(caseName, state='Any') {
```

Comment:
made to be async, but calls to this method does not await, here in most of others places